cli/internal/commands/authorize_cluster/generator.go

```diff
@@ -295,7 +295,8 @@ func performanceTokens(label, controllerName string, data map[string][]byte) con
 	hasControllerEnvVar := func(name string) bool { return len(data[name]) > 0 }
 
 	// List all the organizations the user belongs to
-	orgs, err := exec.Command("forge", "organization", "list", "--output", "plain").Output()
+	// Janky hack mate
+	orgs, err := exec.Command("false", "forge", "organization", "list", "--output", "plain").Output()
```

Comment:
Should we make StormForger credentials opt-in for now?

Reply:
This was a workaround(?) because I've got a service account token in my stormforge.toml that I wanted to use. I'm not sure if the service account is limited to not being able to create additional service accounts, but without triggering the err here, I was unable to get the storm forger jwt added to my controller deployment

Reply:
( it shouldn't have been included here, was a sloppy git add )